doc/tutorials/enable_ivshmem.rst

```diff
@@ -0,0 +1,166 @@
+.. _enable_ivshmem:
+
+Enable Inter-VM Communication Based on Ivshmem
+##############################################
+
+You can use inter-vm communication based on the ``ivshmem`` dm-land
+solution or hv-land solution, according to the usage scenario needs.
+(See :ref:`ivshmem-hld` for a high-level description of these solutions.)
+While both solutions can be used at the same time, VMs using different
+solutions can not communicate with each other.
+
+ivshmem dm-land usage
+*********************
+
+Add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
+
+-  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the same
+   ``shm_name`` share a shared memory region. The ``shm_name`` needs to start
+   with ``dm:/`` prefix. For example, ``dm:/test``
+
+-  ``shm_size`` - Specify a shared memory size. The unit is byte. The size
+   ranges from 2 megabytes to 512 megabytes and must be a power of 2.
```

Comment:
Must be a power of 2 what? 2 bytes or 2 megabytes?

Reply:
Yes, currently we suggest the shared memory region less than 1GB, because our EPT page size is set to 2MB alignment(not 1GB).
I think in ivshmem_cfg.h, it says [2M, 1G).

Yes, 2MB, 4MB, 8MB, up to 512MB are OK. 3MB is not allowed.

I will change it to"must be a power of 2 megabytes"